cmd/report.go

```diff
@@ -34,6 +34,13 @@ var (
 		Use:   "cookbooks",
 		Short: "Generates a cookbook oriented report",
 		Args:  cobra.NoArgs,
+		Long: `Generates cookbook oriented reports containing details about the number of
+violations each cookbook has, which violations can be are auto-corrected and
+the number of nodes using each cookbook.
+
+These reports could take a long time to run depending on the number of cookbooks
+to analyze and therefore reports will be stored in a cache directory (.analyze-cache/).
```

Comment:
Minor suggestion: "... therefore reports will be written to disk. The location will be provided when the report is generated." or similar.  That way if we change this (eg ~/.chef-workstation/analyze/reports or something?) it's one less place to remember updating.

Reply:
Updating....

![tenor-238432266](https://user-images.githubusercontent.com/5712253/70268600-1a802280-17a1-11ea-8cb4-6a932d59533a.gif)